torch/_inductor/codecache.py

```diff
@@ -3177,6 +3137,8 @@ def wait(self, scope: Dict[str, Any]) -> None:
 if (
     os.environ.get("TORCH_TNT_IN_USE", "0") == "1"
     or os.environ.get("TORCH_WARM_POOL", "1") != "1"
+    # TODO(masnesral): warm pool is broken, need to fix it
```

Comment:
@jansel, I commandeered this TODO you had originally, but I'm not sure there's anything to actually do here? We're calling _warm_process_pool() on the ProcessPoolExecutor owned by the subproc in SubprocMain.__init__(). Or did you have some other kind of warmup?

Reply:
I was getting errors with this turned on.  Does it pass for you?